jac-uk/admin
# Allow Users To Raise Bugs On Different Environments

## What's included?
Users can currently only raise bugs on the production environment. These changes should enable them to raise bugs on staging too (and other  platforms potentially).

Use the environment and the platform in the bug reference number.
Use timestamp in reference number if on the local environment to prevent clashes. Only show the feedback link if the environment and platform are defined in the env file.

Closes jac-uk/digital-platform/issues#1030

## Who should test?
✅ Product owner
✅ Developers

## How to test?
Deploy the change to staging and ensure bugs can be raised on both staging and production.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
